protocol/x/clob/keeper/process_operations.go

```diff
@@ -532,6 +540,15 @@ func (k Keeper) PersistMatchLiquidationToState(
 		return err
 	}
 
+	err = k.statefulValidateMatchTaker(ctx, takerOrder.MustGetOrder())
```

Comment:
For my understanding why doesn't this panic for liquidation orders?

Doesn't it call [this function](https://github.com/dydxprotocol/v4-chain/blob/76ef1aefc1bc7ab393c20512e0940ea2be018cdc/protocol/x/clob/types/liquidation_order.go#L100-L104)?

Reply:
It does panic and was causing test failures. We don't need to statefully validate the taker for liquidations (because it isn't a real order). Will remove this validation.